### ISSUE TITLE
3.0.0-beta20 breaks compatibility with nodejs <= 0.12

### ISSUE THREAD

**Issue by rodrigoaguilera** [kss-node/kss-node#478]
I'm just a person maintaining a build system for a project that doesn't fix the versions of the packages we install from npm. This system uses nodejs 0.12 for legacy reasons.

The latest version of kss was always picked but I noticed it breaks the build because it downloads fs-extra@6.0.0 intead of fs-extra@5.0.0 (this latest version is still compatible).

Is there a way to declare this incompatibility in the package.json?

**Comment by RobLoach** [comment]
Good call, thanks... We likely should up the version number to make it more clear too.

**Comment by ryuran** [comment]
https://github.com/kss-node/kss-node/blob/master/package.json#L57-L59

I think this line is enough but I see:
<img width="987" alt="capture d ecran 2018-05-21 a 13 29 08" src="https://user-images.githubusercontent.com/1309340/40305532-3065f118-5cfb-11e8-9d62-6102c2ea09c5.png">

So we should be more clear.

**Comment by DanielRuf** [comment]
Using `engines.node` is mostly sufficient. Or is something different needed?
